tool/release/createnotes/Common.kt

```diff
@@ -10,7 +10,6 @@ import java.nio.file.Path
 
 object Constant {
     val releaseTemplateRegex = "\\{\\s*release notes\\s*}".toRegex()
-    const val installInstruction = "http://docs.vaticle.com/docs/running-typedb/install-and-run"
     const val github = "https://api.github.com"
     const val headerAccept = "\"application/vnd.github.v3+json"
```

Comment:
@lolski quote at the beginning is suspicious - is it supposed to be there and why?

Reply:
Yes I agree! Good spot. Fixed in 3be6d1dd53f720c2490e05d2d4d2e77b078fdac1.